specs/login.spec

```diff
@@ -0,0 +1,6 @@
+# Login in amazon
```

Comment:
what is this file for?

Reply:
It is to run step implementations in gauge. Is this really not required?

Reply:
@anjaliahlawat 
As per spec, I can't see that you are visiting amazon.com and trying to login. These steps are not understandable.